docs/source/tutorials/playing-command.rst

```diff
@@ -0,0 +1,150 @@
+always what you want. Maybe you want the bot to say this::
+
+    <YourNick> .hello
+    <Sopel> Hello YourNick, have a nice day!
+
+For that, you need **the second argument**: the ``trigger``. It is an object
+with all the information you need about the message that triggered the
+callable, such as the **message** itself, the **channel**, the type of message,
+etc. and what we need for now is the
+:attr:`trigger.nick <sopel.trigger.Trigger.nick>` attribute::
+
+    from sopel import plugin
+
+    @plugin.command('hello')
+    def hello(bot, trigger):
+        """Say Hello <user>, have a nice day!"""
+        bot.say(f'Hello {trigger.nick}, have a nice day!')
+
+.. important::
+
+    If you want to test this with your bot, and your bot is already running,
+    you need to restart the bot so it will load the new version of your plugin.
+
+.. seealso::
+
+    You can learn much more about the :class:`trigger <sopel.trigger.Trigger>`
+    object by reading its documentation.
+
+
+Command with arguments
+======================
+
+The trigger object can do much more for you: if a user adds arguments to the
+command, like ``.hello morning``, you can detect and use that argument::
+
+    from sopel import plugin
+
+    @plugin.command('hello')
+    def hello(bot, trigger):
+        """Say Hello <user>, have a nice day!"""
+        # groups 3 to 6 are the first, second, third, and fourth command arg
+        when = trigger.group(3)
+        # select a different greeting depending on when
+        greeting = {
+            'morning': 'and good morning!',
+            'noon': 'are you having lunch?',
+            'night': 'I hope it was a good day!',
+            'evening': 'good evening to you!'
+        }.get(when, 'have a nice day!')  # default to "nice day"
+        # say hello
+        bot.say(f'Hello {trigger.nick}, {greeting}')
+
+Now the command will be able to react a bit more to your user::
+
+    <YourNick> .hello morning
+    <Sopel> Hello YourNick, and good morning!
+    <YourNick> .hello noon
+    <Sopel> Hello YourNick, are you having lunch?
+
+How does that work? Well, the short version is that Sopel uses regex
+(`Regular Expression`__) to match a message to a plugin callable, and the
+``trigger`` object exposes the match result.
+
+.. seealso::
+
+    You can learn much more about the :class:`~sopel.plugin.command` decorator
+    by reading its documentation.
+
+.. note::
+
+    In the case of a command, the regex is entirely managed by Sopel itself,
+    while the generic :func:`@plugin.rule <sopel.plugin.rule>` decorator
+    allow you to define your own regex.
+
+.. __: https://en.wikipedia.org/wiki/Regular_expression
+
+
+And... action!
+==============
+
+Some users say ``.hello`` out loud, and others will say it with an action. How
+do you react to these? Let's go back to our initial example of the ``shrug``
+command: it uses a second decorator, ``action_command('shrugs')``, with a
+different name. How does that work?
```

Comment:
Nit: this command was alluded to, but calling it `our initial example` seems a little bit off to me.

```suggestion
Some users say ``.hello`` out loud, and others will say it with an action. How
do you react to these? Let's go back to the example of the ``shrug`` command:
it uses a second decorator, ``action_command('shrugs')``, with a different
name. How does that work?
```

Reply:
I can see how someone could be confused, so better be safe here. Good call!